src/vs/workbench/contrib/relauncher/browser/relauncher.contribution.ts

```diff
@@ -258,6 +261,50 @@ export class WorkspaceChangeExtHostRelauncher extends Disposable implements IWor
 	}
 }
 
+export class RuntimeArgumentsChangeRelauncher extends Disposable implements IWorkbenchContribution {
+
+	private readonly disableChromiumSandbox = new ChangeObserver('boolean');
+
+	constructor(
+		@IHostService private readonly hostService: IHostService,
+		@IFileService private readonly fileService: IFileService,
+		@IEnvironmentService private readonly environmentService: IEnvironmentService,
+		@IProductService private readonly productService: IProductService,
+		@IDialogService private readonly dialogService: IDialogService
+	) {
+		super();
+
+		this._register(this.fileService.watch(this.environmentService.argvResource));
+		this._register(this.fileService.onDidRunOperation((e) => {
```

Comment:
This will only cover changes to the file done from within VS Code, not from other processes. I think that is actually OK, and as such you could remove the line above that installs a file watcher.